resources/netregexes.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+import Regexes from './regexes.js';
```

Comment:
I think don't trailing `.js`, `import Regexes from './regexes';` is ok enough?
Because quisquous said we're going to migrate cactbot to TypeScript?

Reply:
So, from my testing, not having the `.js` was okay from Webpack's point of view, but not okay from a strict HTML point of view, if the source type was module. Ultimately, I decided to follow:
https://google.github.io/styleguide/jsguide.html#es-module-imports
Since generally we're using the Google style guide, but I don't know how strictly we're following it.

Reply:
I think for now this is fine.  I'd like to move slowly with figuring out to handle the couple of folks who are building things on top of cactbot outside of the repository.  For now, I think keeping these files as directly importable is good.  I think when we move to typescript, there'll need to be some other changes as well and we can change this at that point.  (Or change all imports at that point, if it makes sense).